source/hot-backup.rst

```diff
@@ -24,39 +24,114 @@ To take a hot backup of the database in your current ``dbpath``, do the followin
 
 - Run the ``createBackup`` command as administrator on the ``admin`` database and specify the backup directory. 
 
-  .. code-block:: text
+  .. code-block:: javascript
 
      > use admin
      switched to db admin
-     > db.runCommand({createBackup: 1, backupDir: <backup_data_path})
+     > db.runCommand({createBackup: 1, backupDir: "<backup_data_path>"})
      { "ok" : 1 }
 
 If the backup was successful, you should receive an ``{ "ok" : 1 }`` object.
 If there was an error, you will receive a failing ``ok`` status
 with the error message, for example:
 
-.. code-block:: text
+.. code-block:: javascript
 
    > db.runCommand({createBackup: 1, backupDir: ""})
    { "ok" : 0, "errmsg" : "Destination path must be absolute" }
 
-
 Saving a backup to a TAR archive
 ================================================================================
  
 .. admonition:: Implementation details
  
    This feature was implemented in |PSMDB| 4.2.1-1.
  
- To save a backup in the format of *tar* archive, use the *archive* field to
- specify the destination path:
+To save a backup as a *tar* archive, use the *archive* field to
+specify the destination path:
  
-.. code-block:: text
+.. code-block:: javascript
  
    > use admin
    ...
    > db.runCommand({createBackup: 1, archive: <path_to_archive>.tar })
 
+View backup status
+==================
+
+As of version 4.2.15-16, you can view the backup status. Run the `currentOp <https://docs.mongodb.com/manual/reference/command/currentOp/#mongodb-dbcommand-dbcmd.currentOp>`_ command or use the `$currentOp <https://docs.mongodb.com/manual/reference/operator/aggregation/currentOp/>`_ aggregation stage against the ``admin`` database. 
+
+Example of running the ``currentOp`` command:
+
+.. code-block:: javascript
+
+   > use admin
+   ...
+   > db.runCommand( { currentOp: 1, "$ownOps": 1 } )
+
+Example of running the ``$currentOp`` aggregation stage:
+
+.. code-block:: javascript
+
+   db.getSiblingDB("admin").aggregate([
+      {$currentOp: {} }
```

Comment:
Querying just the backup op in on statement is possible:
```suggestion
      {$currentOp: {} }, 
      { $match: {"command.createBackup": {$exists: true}} }
```

Just in case you're wondering what can be done about the $backupCursor case that should be a match on the field "command.pipeline.$backupCursor".

`{ $match: {"command.pipeline.$backupCursor": {$exists: true}}`